unic/char/range/src/lib.rs

```diff
@@ -0,0 +1,33 @@
+//! # Unic - Char - Range
+//!
+//! A simple way to control iteration over a range of characters.
+//!
+//! # Examples
+//!
+//! ```
+//! # #[macro_use] extern crate unic_char_range;
+//! # use unic_char_range::*;
+//!
+//! # fn main() {
+//! for character in chars!('a'=..='z') {
+//!     // character is each character in the lowercase english alphabet in order
+//! }
+//!
+//! for character in CharRange::all() {
+//!     // character is every valid char from lowest codepoint to highest
+//! }
+//! # }
+//! ```
+//!
+#![forbid(bad_style, missing_debug_implementations, unconditional_recursion)]
+#![deny(missing_docs, unsafe_code, unused, future_incompatible)]
```

Comment:
d24057a for `future_incompatible`. (It doesn't work on 1.17 for some reason.) `unused`, just because I was `allow`ing it temporarily while building the library up.


Basically, I picked `deny` for things that could plausibly be temporarily allowed.